src/components/shared/FormTextArea/index.jsx

```diff
@@ -56,6 +61,11 @@ function TextAreaInput({
         placeholder={placeholder}
         rounded={rounded}
         defaultValue={defaultValue}
+        maxLength={limit}
+        onChange={({ target }) => {
+          onChange();
+          setCharactersCount(target.value.length);
+        }}
```

Comment:
The event could be needed in the parent component in the future, but it looks good to me you can leave it as is. Thanks!

```suggestion
        onChange={(event) => {
          onChange(event);
          setCharactersCount(event.target.value.length);
        }}
```